src/GraphQL.NewtonsoftJson/GraphQLSerializer.cs

```diff
@@ -152,5 +153,62 @@ public async Task WriteAsync<T>(Stream stream, T value, CancellationToken cancel
             _serializer.Serialize(jsonWriter, value);
             await jsonWriter.FlushAsync(cancellationToken).ConfigureAwait(false);
         }
+
+        /// <inheritdoc cref="IGraphQLTextSerializer.Serialize{T}(T)"/>
```

Comment:
Read this comment. It confuses.

Reply:
Updated; please review (also `Read<T>(TextReader)`)